src/Data/Str.php

```diff
@@ -63,11 +63,11 @@ public static function reverse(string $str, $encoding = null) :string
      * Concat the strings.
      *
      * @param string $g   With concat.
-     * @param string $str String to concat.
+     * @param array  $str String to concat.
      *
      * @since 3.0.0
      *
-     * @return bool
+     * @return string
```

Comment:
The `$str` param type is `array` because this method uses `...` syntax.

The sample code for `...` syntax is as follows:

![image](https://user-images.githubusercontent.com/9021747/65939193-bdda5180-e457-11e9-9e74-5e2a85c9c163.png)